tests/Http/Three/SignupTest.php

```diff
@@ -88,19 +91,15 @@ public function testNotCreatingDuplicateSignups()
      */
     public function testUnauthenticatedUserCreatingASignup()
     {
-        $northstarId = '54fa272b469c64d7068b456a';
-        $campaignId = '6LQzMvDNQcYQYwso8qSkQ8';
-        $campaignRunId = $this->faker->randomNumber(4);
-
         $response = $this->json('POST', 'api/v3/signups', [
```

Comment:
oh @DFurnes found another! (sorry for being so nitpicky not sure if this matters too much!)

Reply:
OMG HOW DID I MISS THAT ONE